java/rocksjni/portal.h

```diff
@@ -2381,6 +2381,28 @@ class JniUtil {
     op(key_slice, value_slice);
   }
 
+  static void kv_op_direct_addr(JNIEnv* env, jlong jkey_addr, jint jkey_len,
```

Comment:
@merlimat If it costs noticeable performance to align this method with the other kv_op methods, instead name the helper differently; I found the name didn't align with what the method is doing. How about `kv_create_slices_from_direct_addr()` ?

Reply:
Changing the name sounds good. 

In general, I would avoid the lambda in all these methods to not have the extra allocation. For this PR we shouldn't touch the other methods though.

Reply:
@merlimat FYI when you have the chance to finish up the PR, we will need to get someone on the core RocksDB team to review and merge; while @adamretter and I have a role w.r.t. the Java API, we don't have that power.

Reply:
Hi @merlimat do you plan to update the PR so that the RocksDB core team can review/merge ?

Reply:
@alanpaxton Yes, I'll try to finish up the task in next few weeks.

Reply:
Hi @merlimat (CC @alanpaxton) I am just wondering if there is any update with this?

Reply:
Hi @merlimat are these changes still something you still hope to progress ?